intlist.c

```diff
@@ -0,0 +1,269 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+
+struct node {
+   int value;
+   struct node* link;
+};
+
+//Defines head pointer and fixes it to null

```

Comment:
Use /// < summary>< /summary> comments for method descriptions. This way the user can see your description if they hover over it. 
Use // comments to describe your code wherever needed